src/styles/WishlistScreen/Screen_Wishlist.jsx

```diff
@@ -0,0 +1,45 @@
+import { StyleSheet, Dimensions } from "react-native";
+import Constants from "expo-constants";
+import { COLORS } from "../colors";
+
+export const containers = StyleSheet.create({
+  bgContainer: {
+    /* marginTop: Constants.statusBarHeight, */
```

Comment:
do you need this line?

Reply:
It was just a test, I forgot to remove it, thanks for pointing it out.